crimsobot/cogs/image.py

```diff
@@ -70,7 +101,18 @@ async def bless(self, ctx: commands.Context) -> None:
     @commands.command(brief='Boop the snoot! Must mention someone to boop.')
     async def boop(self, ctx: commands.Context, mention: discord.Member) -> None:
         fp = imagetools.make_boop_img(ctx.author.display_name, mention.display_name)
-        await ctx.send(file=discord.File(fp, 'boop.jpg'))
+
```

Comment:
A little WET here as `boop()` does not go thru `process_image()`. Did not seem worth the effort to try to make this work with the new `get_image_and_embed()` for only this one command.